src/transformState.js

```diff
@@ -6,6 +6,29 @@
  */
 function transformState(state, actions) {
   // write code here
+  const resObj = state;
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
Use 'for ... of' loop instead of 'for' loop to iterate over the 'actions' array.